types/index.d.ts

```diff
@@ -42,6 +42,11 @@ declare namespace RamdaAdjunct {
     type DictPred<T> = (value: T, key: string) => boolean;
 
     interface Static {
+        /**
+         * Invokes the method at path of object with given arguments.
+         */
+        invokeArgs(pathToMethod: string[], args: Array<string | number>, obj: object): any;
```

Comment:
Missing curried form of typings. For more info about how to create one check https://github.com/char0n/ramda-adjunct/blob/54256f34da171b08e0a30b4c198aa697693d1662/types/index.d.ts#L1347